packages/playground/src/components/Header.tsx

```diff
@@ -0,0 +1,197 @@
+import { useCallback } from 'react';
+import Form, { IChangeEvent } from '@rjsf/core';
+import { RJSFSchema, UiSchema, ValidatorType } from '@rjsf/utils';
+import localValidator from '@rjsf/validator-ajv8';
+
+import CopyLink from './CopyLink';
+import ThemeSelector, { ThemesType } from './ThemeSelector';
+import Selector from './Selector';
+import ValidatorSelector from './ValidatorSelector';
+import SubthemeSelector from './SubthemeSelector';
+import RawValidatorTest from './RawValidatorTest';
+
+export interface LiveSettings {
+  showErrorList: false | 'top' | 'bottom';
+  [key: string]: any;
+}
```

Comment:
Should we move this right before HeaderProps where it's being used?

Reply:
That is a personal preference. I just put it where I put it for the convenience of making a single suggestion